assets/js/payplug-apple-pay.js

```diff
@@ -19,23 +20,45 @@
 				$('*[name=woocommerce_checkout_place_order]').show();
 		},
 		ProcessCheckout: function (e) {
+			const request = {
+				"countryCode": apple_pay_params.countryCode,
+				"currencyCode": apple_pay_params.currencyCode,
+				"merchantCapabilities": [
+					"supports3DS"
```

Comment:
what do you think, could this data be at server side? or should it be hardcoded here?

Reply:
Yes `apple_pay_params.countryCode` is coming from the serverside using `wp_localize_script`
I think it may cause a problem in the payment if it is not accurate, but I think `apple_pay_params.currencyCode` is always EUR since it is only for payplug plugin.

Reply:
I was talking about the request data, not only the apple_pay_params